tasks/04_create_links_to_borg_and_borgmatic.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create borgmatic command in /usr/local/bin
+  ansible.builtin.copy:
+    content: |
+      #!/bin/bash
+      . "{{ borg_venv_path }}"/bin/activate
+      borgmatic "$@"
+    dest: /usr/local/bin/borgmatic
+    mode: "0755"
+    owner: "{{ borgbackup_user }}"
+    group: "{{ borgbackup_group }}"
+
+- name: Create borg command in /usr/local/bin
+  ansible.builtin.copy:
+    content: |
+      #!/bin/bash
+      . "{{ borg_venv_path }}"/bin/activate
+      borg "$@"
+    dest: /usr/local/bin/borg
+    mode: "0755"
+    owner: "{{ borgbackup_user }}"
+    group: "{{ borgbackup_group }}"
+
+- name: Install helper scripts
```

Comment:
Maybe this should be optional? Most people won't use your script.

Reply:
Or outside the scope of this role.

Reply:
I would like to make this optional: I need the attached script to backup my Docker containers. In the next pull request will come another one for lxd. I have seen a lot of fiddling and so far nothing that convinced me. If you look at my script, you will see that I have thought of many issues, such as order shutdown and startup etc. I know of no repo where I can leave that, so that all other users again develop their own solution.
My hope is that there are a few other users have an eye on it, to develop this further.

Reply:
Done by TAG